samples/subsys/smf/psicc2/README.rst

```diff
@@ -0,0 +1,51 @@
+.. zephyr:code-sample:: psicc2
+   :name: PSiCC2 State Machine Demo
```

Comment:
to help with discoverability, as the PSiCC2 acronym is rather cryptic, maybe:

```suggestion
   :name: Hierarchical State Machine Demo
```

Reply:
Fair enough, But I'd like to have it reference PSiCC2, since that is the demo name.

Reply:
> Fair enough, But I'd like to have it reference PSiCC2, since that is the demo name.

Ya I know, I was thinking the same, and was wondering if psicc2 really is the best name for this demo/sample (vs. a simpler "hsm" or "hierarchical_state_machine") - again, for the sake of discoverability/clarity. But I also appreciate how this could be annoying to rename at this point.

Reply:
Good point. It is in the SMF folder, so people should know it's a state machine.

I don't mind changing it if people would prefer it, but since Miro is kindly letting us use his example, I feel it should have a link to that. `hsm_psicc2`, maybe?

Reply:
Your suggested name would be great IMO - thanks!

Reply:
OK, done!